package.json

```diff
@@ -23,6 +23,9 @@
   "devDependencies": {
     "ava": "^0.22.0",
     "coveralls": "^2.13.1",
+    "eslint": "^6.8.0",
```

Comment:
I assume these were added by accident

Reply:
Actually they are VSCode eslinter development dependency. What is your recommendation, when to add them?

Reply:
I'd rather leave them out, not to be too opinioned / dependent on particular dev IDE/tooling. The other dev dependencies are for the unit tests and code coverage, and they're part of the CI.

Reply:
(Don't have a very strong opinion though.)